tests/release_tests/test_launch/launch-agent.yml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
```

Comment:
We still don't want to make a new manifest for testing. We should use the existing ones, or as a follow up the helm chart.